src/main/java/thePackmaster/cards/startuppack/BattlePrep.java

```diff
@@ -0,0 +1,44 @@
+package thePackmaster.cards.startuppack;
+
+import basemod.cardmods.InnateMod;
+import basemod.helpers.CardModifierManager;
+import com.evacipated.cardcrawl.mod.stslib.StSLib;
+import com.evacipated.cardcrawl.mod.stslib.actions.common.SelectCardsInHandAction;
+import com.evacipated.cardcrawl.mod.stslib.fields.cards.AbstractCard.FleetingField;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import com.megacrit.cardcrawl.vfx.UpgradeShineEffect;
+import thePackmaster.cards.AbstractPackmasterCard;
+
+import static thePackmaster.SpireAnniversary5Mod.makeID;
+import static thePackmaster.util.Wiz.atb;
+
+public class BattlePrep extends AbstractStartUpCard {
+    public final static String ID = makeID("BattlePrep");
+
+    public BattlePrep() {
+        super(ID, 2, CardType.SKILL, CardRarity.RARE, CardTarget.SELF);
+        FleetingField.fleeting.set(this, true);
+    }
+
+    @Override
+    public void use(AbstractPlayer abstractPlayer, AbstractMonster abstractMonster) {
+        atb(new SelectCardsInHandAction("Make Innate (Battle Prep)", (cards) -> {
```

Comment:
Please don't hardcode strings like this. If this mod ever gets translated, these wouldn't be.
Use extended description string of this card instead